src/olympia/zadmin/views.py

```diff
@@ -474,6 +474,8 @@ def email_devs(request):
             qs = qs.exclude(addon__versions__files__jetpack_version=None)
         elif data['recipients'] == 'all_extensions':
             qs = qs.filter(addon__type=amo.ADDON_EXTENSION)
+        elif data['recipients'] == 'fxa':
+            qs = qs.filter(user__fxa_id=None)
```

Comment:
I think some user's have an empty string set for some reason. I'm not sure how many though.